src/arrayMethodSort.js

```diff
@@ -4,8 +4,31 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  function compareFunction(a, b) {
+    const aStr = String(a);
+    const bStr = String(b);
+
+    if (aStr > bStr) {
+      return 1;
+    }
+
+    if (aStr < bStr) {
+      return -1;
+    }
+
+    return 0;
+  }
+
+  [].__proto__.sort2 = function (customCompareFunction = compareFunction) {
```

Comment:
The parameter 'customCompareFunction' is correctly given a default value of 'compareFunction'. This is a good use of default parameters in ES6. However, it's important to document this behavior for users of your 'sort2' method.